core/rawdb/l2trace_test.go

```diff
@@ -0,0 +1,384 @@
+package rawdb
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/core/types"
+	"github.com/scroll-tech/go-ethereum/rlp"
+)
+
+// TODO reference code
```

Comment:
What's the TODO item here? The text doesn't give much information.

Reply:
The code just for test, can be deleted.